client/src/pages/contribute.js

```diff
@@ -27,19 +29,23 @@ const TableRow = styled(GrommetTableRow)`
   box-shadow: none !important;
 `
 
-const IntakeFormLink = () => (
+const FormLinkButton = ({ ...props }) => (
```

Comment:
Let's explicitly pass in `href` and `label` here and replace the `{...props}` below as well.